src/main/java/com/checkmarx/flow/custom/GitHubIssueTracker.java

```diff
@@ -63,29 +63,41 @@ public void init(ScanRequest request, ScanResults results) throws MachinaExcepti
      */
     @Override
     public List<Issue> getIssues(ScanRequest request) {
-        log.info("Executing getIssues GitHub API call");
+        String apiUrl = String.format("%s/%s/%s/issues?state=all&per_page=%s",
+                properties.getApiUrl(),
+                request.getNamespace(),
+                request.getRepoName(),
+                ISSUES_PER_PAGE);
```

Comment:
Question: if namespace won't be exists in the request, will it still be a valid URL?

Reply:
No, it won't. CxFlow will fail before calling this method with an error: "Namespace / RepoName / Branch are required".